packages/documentation/copy/en/handbook-v2/Everyday Types.md

```diff
@@ -509,7 +509,7 @@ One way to think about this is to consider how JavaScript comes with different w
 
 ```ts twoslash
 let changingString = "Hello World";
-changingString = "Ola Mundo";
+changingString = "Hola Mundo";

```

Comment:
This is/was Portuguese, so not a typo, but now that I see it - it probably needs the diacritic

```suggestion
changingString = "Olá Mundo";
```